ui/src/providers/feast/FeastResourcesContext.js

```diff
@@ -5,28 +5,25 @@ const FeastResourcesContext = React.createContext({});
 
 export const FeastResourcesContextProvider = ({ project, children }) => {
   const [{ data: entities }, listEntities] = useFeastApi(
-    feastEndpoints.listEntities,
-    { method: "POST", muteError: true },
+    `${feastEndpoints.listEntities}?project=${project}`,
+    { method: "GET", muteError: true },
     {},
     false
   );
 
   const [{ data: featureTables }, listFeatureTables] = useFeastApi(
-    feastEndpoints.listFeatureTables,
-    { method: "POST", muteError: true },
+    `${feastEndpoints.listFeatureTables}?project=${project}`,
+    { method: "GET", muteError: true },
     {},
     false
   );
 
   useEffect(() => {
+    console.log("project:", project);
```

Comment:
This is probably added by mistake

Reply:
Thanks for pointing this. I've removed it